src/storage-client/src/controller.rs

```diff
@@ -209,8 +210,11 @@ pub trait StorageController: Debug {
     /// This method can be invoked immediately, at the potential expense of performance.
     fn initialization_complete(&mut self);
 
-    /// Update storage configuration.
-    fn update_configuration(&mut self, config_params: StorageParameters);
+    /// Update storage configuration with new parameters.
+    fn update_parameters(&mut self, config_params: StorageParameters);
+
+    /// Get the current configuration, including parameters updated with `update_parameters`.
+    fn current_configuration(&self) -> &StorageConfiguration;
```

Comment:
The doc comment here makes it clear why you've called this `current_configuration`, but wondering if we could just call this `config`? Do you think there's a sense that this might return some kind of default value?

Reply:
I think `.config()` works!